app/src/main/java/org/mozilla/focus/widget/Cursor.java

```diff
@@ -50,14 +54,23 @@ public void run() {
     };
 
 
+    private Handler cursorHandler = new Handler();
+    private Runnable cursorTick = new Runnable() {
+        @Override
+        public void run() {
+            animate().alpha(0f).setDuration(CURSOR_ANIMATION_DURATION);
```

Comment:
Nit: since we're mixing alphas here (the alphas of the Paint and View, which I think is Android is really bad at perf-wise), I wonder if we want to just "setVisibility(GONE/VISIBLE)". Are people going to see this animation? (Is it too jerky?)